exercises/wordy/wordy_test.rb

```diff
@@ -3,95 +3,127 @@
 require 'minitest/autorun'
 require_relative 'wordy'
 
-class WordProblemTest < Minitest::Test
-  def test_add_1
-    assert_equal 2, WordProblem.new('What is 1 plus 1?').answer
+# Test data version:
+# aa12f2e
+class WordyTest < Minitest::Test
+  def test_addition
+    # skip
+    question = 'What is 1 plus 1?'
+    assert_equal 2, WordProblem.new(question).answer
   end
 
-  def test_add_2
+  def test_more_addition
     skip
-    assert_equal 55, WordProblem.new('What is 53 plus 2?').answer
+    question = 'What is 53 plus 2?'
+    assert_equal 55, WordProblem.new(question).answer
   end
 
-  def test_add_negative_numbers
+  def test_addition_with_negative_numbers
     skip
-    assert_equal(-11, WordProblem.new('What is -1 plus -10?').answer)
+    question = 'What is -1 plus -10?'
+    assert_equal(-11, WordProblem.new(question).answer)
```

Comment:
I think that's a Rubocop warning, it's not particularly important, but it's better to make all the cases consistent so in this case I'd wrap them all in parentheses. This has the pleasant side effect of making your `value_assertion` method much simpler.